components/eam/bld/namelist_files/namelist_definition.xml

```diff
@@ -4250,29 +4250,16 @@ Full pathname of IOP dataset.
 Default: set by build-namelist.
 </entry>
 
-<entry id="scm_crm_mode" type="logical" category="scam"
-       group="cam_inparm" valid_values="">
-Column radiation mode.
-Default: FALSE
-</entry>
-
-<entry id="scm_diurnal_avg" type="logical" category="scam"
-       group="cam_inparm" valid_values="">
-If using diurnal averaging or not.
-Default: FALSE
-</entry>
-
 <entry id="scm_iop_srf_prop" type="logical" category="scam"
        group="cam_inparm" valid_values="">
 Use the specified surface properties.
 Default: FALSE
 </entry>
 
-<entry id="scm_clubb_iop_name" type="char*200" category="scam"
+<entry id="iop_nudge_tscale" type="real" category="scam"
        group="cam_inparm" valid_values="">
-IOP name for CLUBB running in single column mode
-Default: ""
-</entry>
+Time scale to nudge to IOP forcing file.
```

Comment:
general comment about the use of the "IOP" terminology - it seems replacing this with "SCM" would be more general. I have a branch to add an SCM-RCE compset, and obviously an "intense observing period" doesn't make sense in that context. It's not critical that we replace the "IOP" terminology, but I think I would prefer avoiding it if the change was trivial (which it's not).